en/developer_guide/entity_list_ajax_pagination.rst

```diff
@@ -266,6 +266,49 @@ To specify another folder, you must change the configuration :
                         namespace_prefix: "My\\AppBundle"
                         path: @AppBundle/Ressources/config/mymapping
 
+Custom search field of columns
+------------------------------
+
+By default, Open Orchestra provides differents search field for columns (text, date, number and boolean). You can specify the types of fields in data attribute ``input-header`` in link of navigation panel (futher information in document of `navigation panel`_).
```

Comment:
You should illustrate this with a basic example including 2 or 3 fields and showing what to add and where